src/ui/HtmlSettings.tsx

```diff
@@ -1,112 +1,192 @@
 import * as React from 'react';
-import { ButtonGroup } from '@chakra-ui/react';
+import { ButtonGroup, Heading, Stack, Text } from '@chakra-ui/react';
 import { HtmlNavigator, ReaderState } from '../types';
 import Button from './Button';
+import {
+  Continuous,
+  Day,
+  EnlargeFont,
+  Night,
+  Paginated,
+  ReduceFont,
+  Reset,
+  Sepia,
+} from './icons';
+import { FONT_DETAILS } from '../constants';
 import ToggleButton from './ToggleButton';
 import ToggleGroup from './ToggleGroup';
 
 export type HtmlSettingsProps = {
+  buttonTextColor: string;
+  checkedButtonBgColor: string;
```

Comment:
I'm actually not using these values in PdfSettings. I couldn't use the useColorModeValue hook in the HtmlSettings component because it is called conditionally. We can chat tomorrow about whether you would accomplish this differently.